src/models/User.ts

```diff
@@ -1,10 +1,12 @@
 import * as mongoose from "mongoose";
+import * as bcrypt from 'bcrypt';
```

Comment:
Ces imports, tu devrais pouvoir les simplifier avec `import bcrypt from 'bcrypt'`

Reply:
Tests fail when simplifying to `import bcrypt from 'bcrypt'`

Reply:
Je pense qu'il te manque ça : https://www.npmjs.com/package/@types/bcrypt

Reply:
le package n'a pas d'export par défault